packages/chord-detect/index.ts

```diff
@@ -19,21 +19,48 @@ const namedSet = (notes: string[]) => {
   return (chroma: number) => pcToName[chroma];
 };
 
-export function detect(source: string[]): string[] {
+export function detect(source: string[], assumePerfectFifth: boolean = false): string[] {
```

Comment:
I'd prefer an `options` object as a second parameter, because it makes possible to extend the API without breaking changes. Something like this:
```ts
type DetectOptions = {
  assumePerfectFifth: boolean
}
export function detect(source: string[], options?: Partial<DetectOptions>) {
  const assumePerfectFifth = options?.assumePerfectFifth === true
  ...
}
````

What do you think?